CHANGELOG.md

```diff
@@ -1,4 +1,59 @@
+* gkehub: added `membership_location` field to `google_gke_hub_feature_membership` resource ([#16536](https://github.com/hashicorp/terraform-provider-google/pull/16536))
+* logging: made the change to aqcuire and update the `google_logging_project_sink` resource that already exists at the desired location. These logging buckets cannot be removed so deleting this resource will remove the bucket config from your terraform state but will leave the logging bucket unchanged. ([#16513](https://github.com/hashicorp/terraform-provider-google/pull/16513))
+* memcache: added `MEMCACHE_1_6_15` as a possible value for `memcache_version` in `google_memcache_instance` resource ([#16531](https://github.com/hashicorp/terraform-provider-google/pull/16531))
+* monitoring: added error message to delete Alert Policies first on 400 response when deleting `google_monitoring_uptime_check_config` resource ([#16535](https://github.com/hashicorp/terraform-provider-google/pull/16535))
+* spanner: added `autoscaling_config` field to `google_spanner_instance` resource ([#16473](https://github.com/hashicorp/terraform-provider-google/pull/16473))
+* workflows: promoted `user_env_vars` field to GA on `google_workflows_workflow` resource ([#16477](https://github.com/hashicorp/terraform-provider-google/pull/16477))
+
+BUG FIXES:
+* compute: changed `external_ipv6_prefix` field to not be output only in `google_compute_subnetwork` resource ([#16480](https://github.com/hashicorp/terraform-provider-google/pull/16480))
+* compute: fixed issue where `google_compute_attached_disk` would produce an error for certain zone configs ([#16484](https://github.com/hashicorp/terraform-provider-google/pull/16484))
+* edgecontaienr: fixed update method of `google_edgecontainer_cluster` resource ([#16490](https://github.com/hashicorp/terraform-provider-google/pull/16490))
+* provider: fixed an issue where universe domains would not overwrite API endpoints ([#16521](https://github.com/hashicorp/terraform-provider-google/pull/16521))
+* resourcemanager: made `data_source_google_project_service` no longer return an error when the service is not enabled ([#16525](https://github.com/hashicorp/terraform-provider-google/pull/16525))
+* sql: `ssl_mode` field is not stored in terraform state if it has never been used in `google_sql_database_instance` resource ([#16486](https://github.com/hashicorp/terraform-provider-google/pull/16486))
+  
+NOTES:
+* dataproc: backfilled `terraform_labels` field for resource `google_dataproc_workflow_template`, so resource recreation won't happen during provider upgrade from `4.x` to `5.7` ([#16517](https://github.com/hashicorp/terraform-provider-google/pull/16517))
+* * provider: backfilled `terraform_labels` field for some immutable resources, so resource recreation won't happen during provider upgrade from `4.X` to `5.7` ([#16518](https://github.com/hashicorp/terraform-provider-google/pull/16518))
+
+## 5.6.0 (Nov 13, 2023)
```

Comment:
The reason that 5.6 release change log shows here is that the 5.6 changelog is merged to the main branch later than Tuesday. It will affect 5.7 release changelog.